requirements.txt

```diff
@@ -24,6 +24,11 @@ SQLAlchemy>=1.0.10,!=1.1.5,!=1.1.6,!=1.1.7,!=1.1.8     # MIT
 six>=1.10.0                                            # MIT
 virtualenv>=14.0.6                                     # MIT
 
+
+# PyACI
+https://github.com/datacenter/pyaci/archive/master.zip
```

Comment:
This should be specified like this:
-e git+https://github.com/datacenter/pyaci.git@master#egg=pyaci